modules/unsupported/process-feature/src/main/java/org/geotools/process/vector/Contours.java

```diff
@@ -45,6 +45,11 @@
 
 public class Contours {
 
+    private enum PointType {
+        Point,
+        MultiPoint
```

Comment:
Is this change related? Or off topic ...

Reply:
I think this is related to a rule that looks for using == to compare strings instead of the equals method.  It seems like a simple boolean flag could be used here instead of strings or an enum.

Reply:
Yep, it's to avoid string comparison with identity equality.